wordpress-mobile/WordPress-Android
# Feature/stock photos editor and analytics

Resolves #7315, resolves #7311, resolves #7501 - This is final stock photos PR before design review, which adds simple analytics and integration with the editor.

To test:

- Open the editor (Aztec only)
- Tap to add media
- Tap the gallery icon
- Choose "Free Photo Library"
- Search stock photos
- Pick a photo
- Tap Add

The photo should be added to the editor.

![2018-03-24 16_44_15](https://user-images.githubusercontent.com/3903757/37868869-40cbb91a-2f84-11e8-9b21-ee663bdc88d5.gif)
